src/SignaturePad.Forms.iOS/SignaturePadRenderer.cs

```diff
@@ -0,0 +1,215 @@
+using System;
+using System.ComponentModel;
+using CoreGraphics;
+using System.Linq;
+using SignaturePad.Forms;
+using SignaturePad.Forms.iOS;
+using UIKit;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform;
+using Xamarin.Forms.Platform.iOS;
+using NativeSignaturePadView = SignaturePad.SignaturePadView;
+using NativePoint = CoreGraphics.CGPoint;
+using Color = Xamarin.Forms.Color;
+using Point = Xamarin.Forms.Point;
+using System.IO;
+using System.Threading.Tasks;
+
+[assembly: ExportRenderer(typeof(SignaturePadView), typeof(SignaturePadRenderer))]
+
+namespace SignaturePad.Forms.iOS
+{
+    public class SignaturePadRenderer : ViewRenderer<SignaturePadView, NativeSignaturePadView>
```

Comment:
Much of this code appears to be copies per platform.  Maybe add a file link for cross platform code with ifdefs.  For methods that are platforms specific add that to another file with a partial SignaturePadRenderer